data/ex_config.yaml

```diff
@@ -1,35 +1,52 @@
 # A modo with multiple elements. Example GIAB.
 
-- id: ex
-  "@type": MODO
-  description: "Example modo for tests"
-  creation_date: "2024-01-17T00:00:00"
-  last_update_date: "2024-01-17T00:00:00"
-  has_assay: assay1
+- metadata:
+    id: ex
+    "@type": MODO
+    description: "Example modo for tests"
+    creation_date: "2024-01-17T00:00:00"
+    last_update_date: "2024-01-17T00:00:00"
+    has_assay: assay1
 
-- id: assay1
-  "@type": Assay
-  name: Assay 1
-  description: Example assay for tests
-  has_sample: sample1
-  omics_type: GENOMICS
+- metadata:
+    id: assay1
+    "@type": Assay
+    name: Assay 1
+    description: Example assay for tests
+    has_sample: sample1
+    omics_type: GENOMICS
 
-- id: demo1
-  "@type": DataEntity
-  name: Demo 1
-  description: Demo CRAM file for tests.
-  data_format: CRAM
-  data_path: data/ex/demo1.cram
-  has_reference: reference1
+- metadata:
+    id: demo1
+    "@type": DataEntity
+    name: Demo 1
+    description: Demo CRAM file for tests.
+    data_format: CRAM
+    data_path: demo1.cram
```

Comment:
It is a nice idea and only a few lines of additional code (at least for `MODO.from_file()`).
For the `cli` it is more tedious, because `data_path` belongs to the required slots when prompting. I would suggest that there we keep it as it is for now?

Reply:
makes sense, tracked in issues https://github.com/sdsc-ordes/modos-api/issues/85